common/modules/gcp-project/main.tf

```diff
@@ -246,6 +263,13 @@ data "google_iam_policy" "combined" {
     ]
   }
 
+  # Google IAM requires a special "invite" workflow for the Owner
+  # role when the account is not part of the Organization. This
+  # comes up when using user@rtf named accounts in the test
+  # Organization. The error might (unhelpfully) look like this,
+  # followed by a bunch of Go structs:
+  #
+  # googleapi: Error 400: Request contains an invalid argument., badRequest
```

Comment:
This is how I worked around this issue: https://github.com/gpii-ops/gpii-infra/pull/440/files#diff-a211e5b31cacb9f269bc221b59ee16a0R194
Perhaps this can be used to avoid this issue.

Reply:
I raised this topic in standup. Stepan says that owner is not exactly equal to editor + SA admin, so I left the implementation unchanged and kept the comment.

Reply:
See also: https://github.com/gpii-ops/gpii-infra/pull/452